frontend/src/components/users/Grid.tsx

```diff
@@ -46,6 +46,11 @@ export const Grid = ({ widgets }: { widgets: Widgets }) => {
       widgets[index].pos = widgets[index].pos.add(cursorPosition)
       return
     }
+    //push할 수 있는 경우
+    if (isPushable(widgets[index], cursorPosition, widgets) !== false) {
```

Comment:
```suggestion
    if (isPushable(widgets[index], cursorPosition, widgets)) {
```
`!== false` 는 `=== true` 란 뜻인데 더 넣을 필요는 없을 것 같습니다.

Reply:
반영했습니다!